insights/parsers/sealert.py

```diff
@@ -0,0 +1,117 @@
+"""
+Sealert - command ``/usr/bin/sealert -l "*"``
+=============================================
+"""
+
+from insights import CommandParser
+from insights import parser
+from insights.specs import Specs
+from insights.parsers import SkipException
+
+
+class Report(object):
+
+    def __init__(self):
+        self.lines = []
+
+    def __str__(self):
+        return "\n".join(self.lines).strip()
+
+    def append_line(self, x):
+        self.lines.append(x)
+
+    def lines_stripped(self):
+        """
+        Returns the lines without trailing empty lines
+
+        Returns:
+            list: The lines without empty lines at the end of the list.
+        """
+        ret = []
+        for l in reversed(self.lines):
```

Comment:
Reversing list two times is inefficient and unnecessary. 

This implementation is about 10-15x faster:

```python
def lines_stripped_faster(self):
    """
        Returns the lines without trailing empty lines
        Returns:
            list: The lines without empty lines at the end of the list.
    """
    lines = self.lines[:]
    for index in range(len(self.lines) - 1, -1, -1):
        if lines[index] == '':
            del(lines[index])
        else:
            break
    return lines
```
(not thoroughly tested)

Reply:
That's good!